whois-nrtm4/src/main/java/net/ripe/db/nrtm4/dao/WhoisObjectRepository.java

```diff
@@ -0,0 +1,82 @@
+package net.ripe.db.nrtm4.dao;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.PreparedStatementCallback;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.sql.ResultSet;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicReference;
+
+
+@Repository
+public class WhoisObjectRepository {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    WhoisObjectRepository(@Qualifier("whoisSlaveDataSource") final DataSource dataSource) {
+        this.jdbcTemplate = new JdbcTemplate(dataSource);
+    }
+
+    public Integer findLastSerialId() {
+        return jdbcTemplate.queryForObject(
+            "SELECT MAX(serial_id) FROM serials",
+            (rs, rowNum) -> rs.getInt(1));
+    }
+
+    public List<ObjectData> findLastObjects() {
+        return jdbcTemplate.query(
```

Comment:
rename it to getAllLatestObjects or getAllObjectsFromLast (if last means able name?)?